Exam/CC1/src/exercice3/Dictonary.java

```diff
@@ -0,0 +1,22 @@
+package exercice3;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.Map;
+
+public class Dictonary {
+
+	Map<String, String> list = new HashMap<>();
```

Comment:
Hmmm... Why store the description as `String` when you can store directly `Word` objects??